### PR TITLE
Enhancement 1455 custom actions

### PR DESCRIPTION
**Pull request by andyrozman**
Implementation of custom actions #1455

### CODE REVIEW

File: app/src/main/java/info/nightscout/androidaps/plugins/PumpVirtual/VirtualPumpFragment.java [hunk 1]
```diff
@@ -119,4 +121,32 @@ public void run() {
...
+
+        @Override
+        public String getKey() {
+            return this.name();
+        }
+    }
+
+
+
 }
```
Comment (MilosKozak) [inline app/src/main/java/info/nightscout/androidaps/plugins/PumpVirtual/VirtualPumpFragment.java]: tihs should be probably removed

**Comment by MilosKozak** [comment]
sorry messed up resolving conflict. add missing bracket pls

**Comment by MilosKozak** [comment]
will you repost this PR?

**Comment by andyrozman** [comment]
I think that PR is fixed. Can you check please.

**Pull request merged**